resources/fixtures/FixturesMap.xml

```diff
@@ -280,6 +280,9 @@
     <F n="Botex-DSP-405" m="DSP-405"/>
     <F n="Botex-SP-1500" m="SP-1500"/>
   </M>
+  <M n="Bright">
+    <F n="Bright-XBAR" m="Bright XBAR"/>
```

Comment:
This is wrong. Manufacturer should be "Bright".
Please remove this change and leave this to me. (must be created with the dedicated Python script)

Reply:
Ah, I saw the addition of the FixtureMap entry in some other commit adding a fixture, but that was of course your commit.
Done!